templates/svelte/entry-type/ui/src/{{dna_role_name}}/{{coordinator_zome_manifest.name}}/{{#each entry_type.fields}}{{#if (and linked_from (not (eq linked_from.hash_type 'AgentPubKey') ) )}}{{pascal_case (plural ..¡entry_type.name)}}For{{pascal_case linked_from.name}}.svelte{{¡if}}{{¡each}}.hbs

```diff
@@ -23,17 +23,18 @@ onMount(async () => {
   }
 
   try {
-    hashes = await client.callZome({
+    const records = await client.callZome({
       cap_secret: null,
-      role_name: '{{dna_role_name}}',
+      role_name: '{{../dna_role_name}}',
       zome_name: '{{../coordinator_zome_manifest.name}}',
       fn_name: 'get_{{snake_case (plural ../entry_type.name)}}_for_{{snake_case linked_from.name}}',
       payload: {{camel_case linked_from.singular_arg}}
     });
+    hashes = records.map(r => {{#if (eq referenceable.hash_type "ActionHash")}}r.signed_action.hashed.hash{{else}}(r.signed_action.hashed.content as NewEntryAction).entry_hash{{/if}});
   } catch (e) {
     error = e;
   }
-  loading = true;
+  loading = false;
```

Comment:
I suggest to always set `loading` to `false`, regardless of success or error.
```suggestion
} finally {
  loading = false;
}
```

Reply:
I think the current code still does this. If there's an error, we catch, set the `error` variable, and then set loading = false

Reply:
ah that's true!